data/json/items/tools.json

```diff
@@ -11073,5 +11073,68 @@
     "revert_to": "null",
     "use_action": "CROWBAR",
     "qualities": [["HAMMER", 2], ["HAMMER_FINE", 1]]
+  },  {
+    "id": "l-stick",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "L-stick (off)",
+    "name_plural": "l-sticks (off)",
+    "description": "Produced by the Light corporation, this stylish staff not only functions as a light source, but doubles as a light-weight weapon, due to the superalloy materials it is made of.  The staff uses long-lasting batteries as its power source.", 
+    "price": 5000,
+    "material": ["superalloy", "steel"],
+    "techniques" : ["WBLOCK_1", "RAPID"],
+    "flags": ["DURABLE_MELEE"],	
+    "weight": 1000,
+    "volume": 12,
+    "bashing": 20,
+    "cutting": 0,
+    "to_hit": 4,
+    "category" : "weapons",
+    "max_charges": 1000,
+    "initial_charges": 1000,
```

Comment:
Those initial charges could be lower, it's already crazy good without free 1k batteries.